### ISSUE TITLE
Add mortgage expense as a condition to showing certain benefits in results

### ISSUE THREAD

**Issue by kbottagaro** [Gary-Community-Ventures/benefits-api#410]
Modify benefits below so they appear in results ONLY if a user lists mortgage as an expense (other eligibility rules such as income, county still apply): 

Near-term resources:

- Foreclosure Financial Assistance Program
- Colorado Emergency Mortgage Assistance

**Comment by ivonne-hernandez** [comment]
We have a `has_expense` method on the `screen` that I can use.

**Comment by CalebPena** [comment]
Will look something like: `e.condition(self.screen.has_expense(expenses)`

**Comment by kbottagaro** [comment]
@sydney-devine Additional note for this ticket:

Modify benefits below so they show up ONLY if a user lists either 'rent' OR 'mortgage' as expense (subject to other eligibility requirements): 
- Emergency Assistance (Denver)
- Temporary Rent and Utility Assistance Program (TRUA) (Denver)

**Comment by sydney-devine** [comment]
@ivonne-hernandez ^ this ok to add in?

**Comment by ivonne-hernandez** [comment]
> * Emergency Assistance (Denver)

@kbottagaro is the Emergency Assistance (Denver) different from the Emergency Rental Assistance?

**Comment by kbottagaro** [comment]
Yes, they are different programs. Emergency Assistance (Denver) is
available only to Denver residents. Emergency Rental Assistance is a
statewide program, but FYI, I just got off the phone with someone at the
state, and this will likely change.
